system-variables.md

```diff
@@ -779,6 +758,10 @@ Constraint checking is always performed in place for pessimistic transactions (d
 >
 > - Since v5.4.0, for a newly deployed TiDB cluster, this variable is enabled by default.
 
+- Scope: SESSION | GLOBAL
+- Default value: `ON`
+- This variable is used to control whether to enable the index merge feature.
```

Comment:
This just moves the comment above the feature specification, to be consistent. We generate it this way in every other case.